bin/install_upgrade.escript

```diff
@@ -231,6 +297,10 @@ parse_version(V) when is_list(V) ->
     hd(string:tokens(V,"/")).
 
 check_and_install(TargetNode, Vsn) ->
+    %% Backup the vm.args. VM args should be unchanged during hot upgrade
+    %% but we still backup it here
+    {ok, [[CurrVmArgs]]} = rpc:call(TargetNode, init, get_argument, [vm_args], ?TIMEOUT),
+    {ok, _} = file:copy(CurrVmArgs, filename:join(["releases", Vsn, "vm.args"])),
```

Comment:
this file is never used, no need to backup ?

if we do need it after all, we need to add vm_args back to boot script.